scripts/speak.py

```diff
@@ -14,18 +16,22 @@
     "xi-api-key": cfg.elevenlabs_api_key
 }
 
+mutex_lock = Lock() # Ensure only one sound is played at a time
+queue_semaphore = Semaphore(1) # The amount of sounds to queue before blocking the main thread
+
 def eleven_labs_speech(text, voice_index=0):
     tts_url = "https://api.elevenlabs.io/v1/text-to-speech/{voice_id}".format(
         voice_id=voices[voice_index])
-    formatted_message = {"text": text}
+    formatted_message = {"text": text, "voice_settings": {"stability": 0.05, "similarity_boost": 0.8}}
```

Comment:
How is this related to the blocking? If not submit it apart

Reply:
It is indeed not related to the blocking. I have reverted this change this the latest commit.